exchanges/btse/btse_websocket.go

```diff
@@ -33,7 +33,10 @@ func (b *BTSE) WsConnect() error {
 	if err != nil {
 		return err
 	}
-	go b.Pinger()
+	b.WebsocketConn.SetupPingHandler(wshandler.WebsocketPingHandler{
```

Comment:
just making sure this doesn't need handler.Message

Reply:
Not for this one
From the docs:
>  To ensure that your session remains active, send a Ping frame before you reach the 60 second timeout.

Previous implementation was:
`b.WebsocketConn.Connection.WriteMessage(websocket.PingMessage, nil)`
which is the same as what's happening here